osCam/templates/registration/login.html

```diff
@@ -1,24 +1,27 @@
 <html>
   <head>
-    {% include "core/bootstrap.html" %}
+    {% include "../core/bootstrap.html" %}
     {% load static %}
+    <link rel="stylesheet" href="{% static 'core/login.css' %}">
   </head>
   <body>
     {% include "core/navigation.html" %}
     <div class="container">
       <div class = "jumbotron">
         <h1>Login</h1>
         <form action="/login/" method="POST">{% csrf_token %}
+          <div style="color: red">
```

Comment:
could be given a div id and moved to css file. easy to move in a css cleanup later